.circleci/config.yml

```diff
@@ -8,22 +8,22 @@ workflows:
     jobs:
       - architect/go-build:
           context: architect
-          name: go-build
-          binary: template
+          name: go-build-muster
```

Comment:
```suggestion
          name: go-build
```

Reply:
Don't you think the name makes more sense as it is currently ?